### PR TITLE
Service Meshes update in Adapters section : PR for #5393

### LINKED ISSUE #5393: [Docs] Service Meshes update
#### Current State
Octarine project is shutdown.

#### Desired State
Remove Octarine Service Mesh from the table. here: https://docs.meshery.io/service-meshes

---

#### [Contributor Guides](https://docs.meshery.io/project/contributing) and Resources
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by karnatisrinivas**
**Description**

This PR fixes #5393 

**Notes for Reviewers**
As per my understanding the Octarine project is shutdown and needs to be removed from the adapters list. Kindly review and please let me know if any edits required. 

Screenshot 2022-04-16 at 1 03 26 PM

Screenshot 2022-04-16 at 1 03 02 PM

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 52df0cc1c334c0eb308c992679bc419401f1f907 at: https://625a74166b9d2b5981c45c36--meshery-docs-preview.netlify.app

File: docs/pages/project/contributing/contributing-docs.md [hunk 1]
```diff
@@ -165,7 +165,7 @@ Restart your WSL VM before moving forward.
 If you've Docker and `make` installed in your system, then you can serve the site locally
 
 ```
-$ make docker-docs
+$ make docker
```
Comment (warunicorn19) [inline docs/pages/project/contributing/contributing-docs.md:168]: Why this change?? I dont think we have a `make docker` command, we have a `make docs` command.

Comment (karnatisrinivas) [reply docs/pages/project/contributing/contributing-docs.md:168]: Hello @warunicorn19 , In the meshery Contribution guide for docs. We have steps to follow using Docker as follows,

Screenshot 2022-04-17 at 11 07 24 AM

But when I tried to serve used the command `make docker-docs`, It says "No target defined". So I checked the [Makefile in Docs folder](https://github.com/meshery/meshery/blob/master/docs/Makefile) and [Make file in Root Folder](https://github.com/meshery/meshery/blob/master/Makefile). 
Screenshot 2022-04-17 at 11 05 29 AM
Screenshot 2022-04-17 at 11 05 12 AM

But `docker-docs` wasn't defined at either locations. So I made a commit in this file.

Comment (Chadha93) [reply docs/pages/project/contributing/contributing-docs.md:168]: @karnatisrinivas would you please revert this `docker-docs` change?

**Review by Chadha93 — CHANGES_REQUESTED** [review]
@karnatisrinivas also please remove the entry from the list for Octarine service mesh.

**Comment by l5io** [comment]
🚀 Preview for commit 349c60c054f38d09e6b074095d08f238a22ac3df at: https://625ef1d846421e005fd5dc1b--meshery-docs-preview.netlify.app

File: docs/pages/project/contributing/contributing-docs.md (hunk 1 above)
Comment (karnatisrinivas) [reply docs/pages/project/contributing/contributing-docs.md]: Reverted the commit made to contributing-docs.md

**Comment by l5io** [comment]
🚀 Preview for commit 5f09da19b4e6cb9bef057cde373c581b4bd14b9f at: https://625ef7f5174c3a03139321d7--meshery-docs-preview.netlify.app

**Review by karnatisrinivas — COMMENTED** [review]
Removed Octarine adapter from the navigation list

**Comment by karnatisrinivas** [comment]
Removed Octarine adapter from the navigation list, and reverted a commit. Please review.

**Review by Chadha93 — APPROVED** [review]
LGTM, Thank you @karnatisrinivas

**Review by warunicorn19 — APPROVED** [review]
LGTM, Thank you @karnatisrinivas
